src/FakeItEasy/Configuration/ICallOrderAssertConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+namespace FakeItEasy.Configuration
+{
+    /// <summary>
+    /// Allows the developer to check that an asserted call happened in order.
+    /// </summary>
+    public interface ICallOrderAssertConfiguration

```

Comment:
I don't really like that name... if you can think of something better, let me know